Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,14 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+// My code starts here 
+
+function BMIcalc(w, h) {
```

Comment:
Typically, we write function names like this: verbNoun, or what it _does_ and what it does it to. 

computeValue , convertName, getProperty. What might you name this?

Reply:
Thank you, I will in future write a function name like "calculateBMI" in future :-)
I can see why reading it the way I initially wrote it, if I continued a pattern like that, would be difficult to understand.